tsconfig.json

```diff
@@ -42,11 +42,11 @@
     // "rootDirs": [],                                   /* Allow multiple folders to be treated as one when resolving modules. */
     // "typeRoots": [],                                  /* Specify multiple folders that act like './node_modules/@types'. */
     "types": [
-        "./worker-configuration.d.ts",
-        "./worker-configuration.d.ts",
-        "node",
-        "node",
-        "vite/client"
+      "./worker-configuration.d.ts",
+      "./worker-configuration.d.ts",
```

Comment:
minor question but why do these get doubled?

Reply:
They shouldn't be. Will remove